VirtoCommerce.Platform.Core/Notifications/IEmailSender.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace VirtoCommerce.Platform.Core.Notifications
+{
+    public interface IEmailSender
+    {
+        Task SendEmailAsync(string to, string type, string languageCode, Dictionary<string, object> parameters);
```

Comment:
Use this signature  `public Task SendEmailAsync(string email, string subject, string message)`